### PR TITLE
Add `find_circ`

### PR DESCRIPTION
**Pull request by BarryDigby**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by BarryDigby** [comment]
@BiocondaBot please add label

File: recipes/find_circ/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "find_circ" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marvin-jens/find_circ.git
```
Comment (pinin4fjords) [inline recipes/find_circ/meta.yaml:9]: I can see a couple of releases on this repo- is there a reason [a stable URL](https://bioconda.github.io/contributor/guidelines.html#stable-urls) can't be used?

File: recipes/find_circ/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - python ==2.7.15
+    - pip
+  run:
+    - python ==2.7.15
+    - numpy ==1.16.4
+    - pysam ==0.15.2
+
+test:
+  commands:
+    - python --version
```
Comment (pinin4fjords) [inline recipes/find_circ/meta.yaml:27]: The test needs to be something specific to the package to check it installed correctly, even if it's just the imports like [this](https://github.com/bioconda/bioconda-recipes/blob/57e2c903407676cca776299a9e0bf82036c0631a/recipes/scanpy-scripts/meta.yaml#L73). 

The Python version check won't work for that.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**